dwcj-widgets/dwcj-code-widget/pom.xml

```diff
@@ -0,0 +1,25 @@
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.dwcj</groupId>
+        <artifactId>dwcj-project</artifactId>
+        <version>0.12.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>dwcj-code-widget</artifactId>
```

Comment:
@StephanWald how about we use `dwcj-addons` instead of `dwcj-widgets` to avoid any possible confusion with BBj widgets and plugins

Reply:
Makes sense. I did the renaming, have a look if you like it better now.

Reply:
Much better. Shouldn't we also rename the package names?

`org.dwcj.widgets.code` to `org.dwcj.addons.code`

Reply:
hmm - still see value in distinguishing widgets from other stuff. don't know